packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal-content-wrapper.tsx

```diff
@@ -10,7 +10,8 @@ import JurisdictionModalFootNote from './jurisdiction-modal-foot-note';
 import { useStore, observer } from '@deriv/stores';
 import { useCfdStore } from '../../Stores/Modules/CFD/Helpers/useCfdStores';
 
-const JurisdictionModalContentWrapper = observer(({ openPasswordModal }: TJurisdictionModalContentWrapperProps) => {
+const JurisdictionModalContentWrapper = observer((props: TJurisdictionModalContentWrapperProps) => {
+    const { openPasswordModal } = props;
```

Comment:
@arshad-rao-deriv no need to make this change because we don't use `props` variable in this file  :)

Reply:
Updated.